templates/microsoft-SQLServer2017/README.md

```diff
@@ -0,0 +1,21 @@
+Microsoft SQL Server 2017 Database Monitoring
+=============================================
+
+Monitors the Microsoft SQL Server Database installed on a Windows server.
+
+Provides monitoring of:
+* Logs
+* Data Directory
+* Events
+* Performance metrics.
```

Comment:
There's an unwanted dot on this line :-)

Reply:
Dot removed